src/Sarif.Converters/ConverterResources.resx

```diff
@@ -201,19 +201,10 @@
   <data name="SemmleUnknownSeverity" xml:space="preserve">
     <value>The severity "{0}" does not correspond to a SARIF result level.</value>
   </data>
-  <data name="ErrorMissingConverterType" xml:space="preserve">
-    <value>The converter plugin assembly "{0}" does not contain a public class "{1}".</value>
```

Comment:
Remove several errors: when these things happen, the plugin factory now just returns null and delegates to the next factory in the chain.